projects/observability/src/pages/apis/services/services-list.dashboard.ts

```diff
@@ -19,6 +24,17 @@ export const servicesListDashboard: DashboardDefaultConfiguration = {
           type: 'entity-specification'
         }
       },
+      {
+        type: 'table-widget-column',
+        title: 'Service Name',
+        display: CoreTableCellRendererType.Text,
```

Comment:
Shouldn't this be an entity renderer?

Reply:
If we make this an entity spec that fetches serviceName and serviceId, we'd be able to render this as a full entity supporting drilldown